code/modules/mob/living/simple_animal/hostile/mining_mobs.dm

```diff
@@ -447,7 +447,7 @@
 	w_class = 3
 	layer = 4
 
-/obj/item/asteroid/goliath_hide/afterattack(atom/target, mob/user, proximity_flag)
+/obj/item/stack/sheet/animalhide/goliath_hide/afterattack(atom/target, mob/user, proximity_flag)
```

Comment:
Just replace any 'qdel()' in that proc with 'use(1)'. Then test it. Should work.